content/how-programs-work/reading/your-first-program/index.md

```diff
@@ -0,0 +1,230 @@
+
+## Open your terminal
+
+You will use the terminal to create a location to store the code repos.  The terminal will also communicate with GitHub.
+
+1. Run the following command: `ls`
+This will list any files or directories (folders) at the level of your terminal. 
+
+   It may look like this:
+   ```bash
+   computer:~ username$ ls
+   Applications   
+   Desktop
+   Downloads
+   Documents
+   ...
+   ```
+1. Select a location where you would like to store your course work. The example is storing work on the Desktop, you may choose a different location if you desire.
+
+   1. Use the following command to move into that location: `cd <location-name>`
+      ```bash
+      computer:~ username$ cd Desktop
+      ```
+   1. Create a directory (folder) to save your course work: `mkdir <directory-name>`
+      ```bash
+      computer:~ username$ mkdir LaunchCode
+      ```
+   1. To verify if your directory exists run the `ls` command again.  You should see your new directory listed.
+   1. Move into your new directory with the following command: `cd <directory-name>`
+      ```bash
+      computer:~ username$ cd LaunchCode
+      computer:LaunchCode username:
+      ```
+
+You are now ready to add the repo to your directory.  
+
+
+## Go to GitHub
+
+You will be provided links to all of the repos in the course.  You will need to **fork** and **clone** them to your computer.  When you fork a repo, you are making a copy of the original code.  Any changes you make to the forked repo will not be applied to the original code.  This keeps the original starter code in tact for the next student.  Cloning the repo moves the repo code onto your computer.  If the code is not on your computer, you will not be able to work with it.  
+
+1. Make sure you are signed into your GitHub account.
+1. Go to the starter code repo and fork it.
+   {{< rawhtml >}}
+      <img src="../images/forking-a-repo.png" alt="The starter code repo with the fork button" />
+   {{< /rawhtml >}}
+   A. The original repo.  Notice the Lauchcode rocket next to the name.
+   
+   B. The _Fork_ button.  Click this to create a fork. You should be redirected to your fork once you click this button.
+
+1. Your forked repo will look like this:
+   {{< rawhtml >}}
+      <img src="../images/fork-and-clone.png" alt="GitHub page pointing out that the repo was forked, dropdown menu to clone code is set for HTTPS" />
+   {{< /rawhtml >}}
+   
+   A. The repo will have the same name, but below it will be a link to the original repo.
+
+   B. Click the green _<> Code_ button.  This will open a new dropdown menu.
+
+   C. In the dropdown menu, select _HTTPS_.  A URL will appear. Copy that URL.
+
+
+
+GitHub has documentation on [Forking a repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo#forking-a-repository) and [Cloning your forked repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo) if you would like more guidance. 
```

Comment:
Add "#cloning-your-forked-repository" to the hyperlink "Cloning your forked repository" to go to the right section.

Reply:
Fixed the above issue.